src/hooks/screen/useChatScreenHook.ts

```diff
@@ -73,6 +74,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
 
   const sendChat = async (
     message: string = randomString(20),
+    attachments: [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function now accepts an additional parameter `attachments`. This is a good change as it allows sending attachments along with the chat message.
```diff
+     attachments: [],
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->